libs/blocks/tabs/tabs.js

```diff
@@ -84,13 +87,41 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
-let initCount = 0;
+function handleDeferredImages(e) {
+  const images = e.querySelectorAll('img[loading="lazy"]');
+  images.forEach((img) => {
+    /* c8 ignore next */
+    img.removeAttribute('loading');
+  });
+  e.removeEventListener('milo:deferred', handleDeferredImages, true);
+}
 
 const init = (block) => {
   const rootElem = block.closest('.fragment') || document;
   const rows = block.querySelectorAll(':scope > div');
+  const parentSection = block.closest('.section');
   /* c8 ignore next */
   if (!rows.length) return;
+  block.addEventListener('milo:deferred', handleDeferredImages(block), true);
+
+  // Tab Config
+  const config = {};
+  const configRows = [].slice.call(rows);
+  configRows.splice(0, 1);
+  if (configRows) {
+    configRows.forEach((row) => {
+      console.log('config row', row.children[0].textContent);
```

Comment:
Are these row config logs being used? Maybe they could be deleted?

Reply:
Good eye, No thats was testing some stuff. Removed now